server/src/instant/model/app.clj

```diff
@@ -49,6 +49,22 @@
                   a.created_at < ?"
               creator-id created-before]))))
 
+(defn get-with-creator-by-ids
+  ([params] (get-with-creator-by-ids aurora/conn-pool params))
+  ([conn app-ids]
+   (sql/select conn ["SELECT a.*, u.email AS creator_email
+                      FROM apps a
+                      JOIN instant_users u ON a.creator_id = u.id
+                      WHERE a.id in (select unnest(?::uuid[]))"
+                     (-> app-ids
+                         vec
+                         (with-meta {:pgtype "uuid[]"})
+                         into-array)])))
```

Comment:
not entirely sure why this is all necessary but this is the only way I was able to get this to work :shrug:

Reply:
Haha love the REPL work!

Reply:
I am honestly not sure why it's needed. Once you land can play with it!